docs/pages/management/dynamic-resources/import-terraform-resources.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Import Teleport Resources into Terraform
+description: Describes how to import existing Teleport resources using the Teleport Terraform provider
+---
+
+If you want to use Terraform to manage Teleport dynamic resources, but have
+already created resources with a tool like `tctl`, you can automatically
+generate Terraform configuration for them. This guide shows you import Teleport
+resource into your Terraform project.
+
+## Prerequisites
+
+- The Teleport Terraform provider. Read the [Terraform provider getting started
+  guide](./terraform-provider.mdx) for how to set up Terraform to manage
+  Teleport dynamic resources. This guide assumes that you have set up a root
+  Terraform module for Teleport on your workstation and started `tbot` to to
+  retrieve credentials for the provider.
```

Comment:
We reference the main guide here as a prerequisite, but it isn't immediately clear why the steps in the main guide are not enough or if only _part_ of those steps should be followed.

Reply:
I've suggested merging the import guide into the setup guide. Since I'm no longer including a comprehensive table of resource IDs, the instructions are a lot shorter, and it's easier to incorporate into the setup guide. This also makes the relationship between these sets of instructions clearer.

Reply:
Sounds good, thanks!